docs/src/main/sphinx/connector/base-jdbc.rst

```diff
@@ -0,0 +1,194 @@
+===================
+JDBC connector base
+===================
+
+A number of connectors use JDBC connections to the data source. Typically
+specific JDBC drivers for the relevant data source are used, and numerous
+extensions in the connector ensure that the specifics of the data sources are
+taken into account. They include variations in data type mappings, support for
+specific SQL keywords and functions, and other aspects.
+
+The following connectors use the JDBC connector base:
+
+* :doc:`clickhouse`
+* :doc:`druid`
+* :doc:`mysql`
+* :doc:`oracle`
+* :doc:`postgresql`
+* :doc:`redshift`
+* :doc:`memsql`
+* :doc:`sqlserver`
+
+These connectors share user-facing behavior and features, documented in the
+following sections.
+
+.. _base-jdbc-configuration:
+
+Configuration
+-------------
+
+.. _base-jdbc-catalog-properties:
+
+Standard catalog properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A common JDBC connector implementation includes a catalog file with the
+following properties:
+
+.. code-block:: properties
+
+    # Connector name (required property)
+    connector.name=generic-jdbc
+
+    # Connection URL (required for JDBC connectors)
+    connection-url=[connectorprotocol]//<host>:<port>;database=<database>
+
+    # Connection credentials
+    connection-user=USERNAME
+    connection-password=PASSWORD
+
+While the ``connector.name`` is the only required property in a catalog file,
+JDBC connectors need a specified ``connection-url``. Typically, username and
+password credentials are also required.
+
+Once the catalog properties file is established, you can add any additional
+configuration (config) properties as needed.
```

Comment:
remove (config) and just don't shorten the word anywhere, as it's specifically against the standard (https://developers.google.com/style/word-list)

Reply:
Good catch..